### DISCUSSION TITLE
How to change skip button text

### DISCUSSION THREAD

**Discussion by arvin-kanani** [AppIntro/AppIntro#951] [Q&A] [answered]
Hi Guys,
I have a program in German language,
how can I change the skip button text?

**Comment by cortinico** [comment, marked as answer]
> I have a program in German language,
> how can I change the skip button text?

The library already support German. The back button will be translated to `ZURÜCK`.
https://github.com/AppIntro/AppIntro/blob/0a991ffa07d30ff866b02952c966b6b0e3bbc369/appintro/src/main/res/values-de/strings.xml#L6

If you wish to have a custom value, you can do it by providing a new string resource such as:
```
<string name="app_intro_back_button">Your Custom String Here</string>
```
